packages/performance/src/services/perf_logger.ts

```diff
@@ -85,7 +85,7 @@ interface TraceMetric {
   custom_attributes?: { [key: string]: string };
 }
 
-/* eslint-enble camelcase */
+/* eslint-disable camelcase */
```

Comment:
notable. The obvious change `eslint-enable` results in a lot of failures

Reply:
I hope you mean `/* eslint-enable camelcase */` (for line 249).

Reply:
Hah, yes. Copy pasta.